### ISSUE TITLE
es2015-localforage commented out function?

### ISSUE THREAD

**Issue by PoBuchi** [Meteor-Community-Packages/ground-db#137]
Is any reason why in 

```
invalidate() {
    console.log('invalidate()', this._collection.queries)
    Object.keys(this._collection.queries)
      .forEach(key => {
        // this._collection.queries[key].changed();
      });
  }
```

https://github.com/GroundMeteor/db/blob/es2015-localforage/lib/client/ground.db.js#L286

is commented out? As without this line, update on doc cause error as minimongo don't understands where query disappeared :)

**Comment by PauliBuccini** [comment]
Ok, looks like it doesn't happen all the time, trying to figure out why so

**Comment by raix** [comment]
I've released `2.0.0-alpha.1` [Notes about the new version, migration and what it doesn't do](https://github.com/GroundMeteor/db/issues/153#issuecomment-206125703) - The issue might be resolved in that version, let me know what you think
